qutebrowser/config/configdata.yml

```diff
@@ -48,6 +48,15 @@ search.incremental:
   default: True
   desc: Find text on a page incrementally, renewing the search for each typed character.
 
+search.wrap:
+  type: Bool
+  default: True
+  desc: >-
+    Wrap around at the top and bottom of the page when advancing through text matches
+    using `:search-next` and `:search-prev`.
+
+    When using QtWebEngine, Qt>=5.14 is required to disable wrapping.
```

Comment:
Instead of mentioning it in the documentation, you can use the `backend` key:

```yaml
  backend:
    QtWebEngine: Qt 5.14
    QtWebKit: true
```